### PR TITLE
[Compute] `az vmss create/update`: Add new parameters `--priority-count` and `--priority-percentage` to support setting priority mix policy

### PR DESCRIPTION
**Pull request by PARADISSEEKR**
**Related command**

`az vmss create`
`az vmss update`

**Description**

Add priorityMixPolicy property to VMScaleSets https://github.com/Azure/azure-cli/issues/23671

**Testing Guide**

https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-orchestration-modes

**History Notes**

[Compute] `az vmss create/update`: Add new parameters `--priority-count` and `--priority-percentage` to support setting priority mix policy

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Compute

File: src/azure-cli/azure/cli/command_modules/vm/custom.py [hunk 1]
```diff
@@ -3743,6 +3745,21 @@ def update_vmss(cmd, resource_group_name, name, license_type=None, no_wait=False
             'vTpmEnabled': enable_vtpm
         }}
 
+    if regular_priority_count is not None or regular_priority_percentage is not None:
+        if vmss.orchestration_mode != 'Flexible':
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/custom.py:3749]: > if namespace.orchestration_mode.lower() != flexible_str.lower():

Why not keep consistent with the above verification logic?

File: src/azure-cli/azure/cli/command_modules/vm/_template_builder.py [hunk 2]
```diff
@@ -1257,6 +1258,15 @@ def build_vmss_resource(cmd, name, computer_name_prefix, location, tags, overpro
         if spot_restore_timeout:
             vmss_properties['spotRestorePolicy']['restoreTimeout'] = spot_restore_timeout
 
+    if (regular_priority_count is not None or regular_priority_percentage is not None) and \
+            cmd.supported_api_version(min_api='2022-08-01', operation_group='virtual_machine_scale_sets'):
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/_template_builder.py:1262]: >  cmd.supported_api_version(min_api='2022-08-01', operation_group='virtual_machine_scale_sets'):

We have added the setting of `min_api` to the definition of parameters. Why do we need to add the `cmd.supported_api_version` to the logic?

File: src/azure-cli/azure/cli/command_modules/vm/custom.py (hunk 1 above)
Comment (PARADISSEEKR) [reply src/azure-cli/azure/cli/command_modules/vm/custom.py:3749]: From service error: Specify priorityMixPolicy is allowed only for virtual machine scale sets with orchestration mode 'flexible'

File: src/azure-cli/azure/cli/command_modules/vm/_template_builder.py (hunk 2 above)
Comment (PARADISSEEKR) [reply src/azure-cli/azure/cli/command_modules/vm/_template_builder.py]: Deleted

**Review by zhoxing-ms — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/vm/custom.py (hunk 1 above)
Comment (zhoxing-ms) [reply src/azure-cli/azure/cli/command_modules/vm/custom.py:3749]: I mean, check in lowercase

**Comment by aleclawlor** [comment]
Hi @PARADISSEEKR, I want to verify that this PR will be merged soon and released as part of the 10-12 CLI release. I see the changes have been approved. Thank you for your help.

**Comment by PARADISSEEKR** [comment]
> Hi @PARADISSEEKR, I want to verify that this PR will be merged soon and released as part of the 10-12 CLI release. I see the changes have been approved. Thank you for your help.

Yes it will be released on 10-12

**Pull request merged**